src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/BaseRecalibratorSparkFn.java

```diff
@@ -54,18 +50,13 @@ public static RecalibrationReport apply( final JavaPairRDD<GATKRead, ReadContext
 
         BaseRecalibrationEngine.finalizeRecalibrationTables(combinedTables);
 
-        try {
-            final QuantizationInfo quantizationInfo = new QuantizationInfo(combinedTables, recalArgs.QUANTIZING_LEVELS);
+        final QuantizationInfo quantizationInfo = new QuantizationInfo(combinedTables, recalArgs.QUANTIZING_LEVELS);
 
-            // TODO: we write the report out and read it back in only because the report changes when written
-            // TODO: out and read back in -- remove this step once we fix this issue.
-            File tempReport = IOUtils.createTempFile("temp-recalibrationtable-", ".tmp");
-            try ( PrintStream reportStream = new PrintStream(tempReport) ) {
-                RecalUtils.outputRecalibrationReport(reportStream, recalArgs, quantizationInfo, combinedTables, new StandardCovariateList(recalArgs, header), recalArgs.SORT_BY_ALL_COLUMNS);
-            }
-            return new RecalibrationReport(tempReport);
-        } catch (FileNotFoundException e) {
-            throw new GATKException("can't find my own temporary file", e);
-        }
+        final StandardCovariateList covariates = new StandardCovariateList(recalArgs, header);
+        final RecalibrationArgumentCollection RAC = recalArgs;
```

Comment:
Use `recalArgs` directly -- we've tried hard to purge this cryptic `RAC` acronym from everywhere